### PR TITLE
Admin: home page: fix typo #7257

### LINKED ISSUE #7257: Admin: home page: fix typo
Change from `Add Instructor` to `Add Instructors` is required as the details are expected from multiple Instructors.
screen shot 2017-05-09 at 11 43 15 am

### PR DESCRIPTION
**Pull request by stanislous**
Fixes The label 'add instructor' is changed to 'add instructors' in AdminHomePage.html in #7257.

### CODE REVIEW

File: docs/development.md [hunk 1]
```diff
@@ -170,7 +170,7 @@ It is recommended to use Firefox 46.0 as this is the browser used in CI build (T
 * You need to use chromedriver for testing with Chrome.
   * Download the latest stable chromedriver from [here](https://sites.google.com/a/chromium.org/chromedriver/downloads).
     The site will also inform the versions of Chrome that can be used with the driver.
-  * Specify the path to the chromedriver executable in `test.chromedriver.path` value in `test.properties`.
+  * Specify the path to the chromedriver executable in `test.chromedriver.path` value in `test.template.properties`.
```
Comment (whipermr5) [inline docs/development.md]: Please revert this change as it's unrelated to the issue. Also, I believe you now know [why it's wrong](https://github.com/TEAMMATES/teammates/pull/7572#issuecomment-310674892)?

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,15 @@
+[*]
```
Comment (whipermr5) [inline .editorconfig]: Please revert this entire file, as it's an unrelated change.

File: src/main/resources/userEmailTemplate-pendingCommentsCleared.html [hunk 3]
```diff
@@ -1,26 +0,0 @@
-<p>Hello ${userName},</p>
```
Comment (whipermr5) [inline src/main/resources/userEmailTemplate-pendingCommentsCleared.html]: This file too.

Comment (stanislous) [inline src/main/resources/userEmailTemplate-pendingCommentsCleared.html]: I reverted above files and changed only the .tag file.

Comment (whipermr5) [inline src/main/resources/userEmailTemplate-pendingCommentsCleared.html]: Remember to push :)

**Comment by stanislous** [comment]
@whipermr5 thank you very much for helping me. I open a new pull request and push my changes. But still, all checks are failed. Can you explain to me what should I do?

**Comment by whipermr5** [comment]
@stanislous No problem :) You can click on the cross to see the build log. You'll need to update one or more test expected HTML files since the output HTML has changed. You may also want to take a look at `GodMode.md` in the docs.
